07-reputation/04-globaltrust.md

```diff
@@ -0,0 +1,39 @@
+
+Each manager retrieves the local trust value of its child at the end of each epoch and stores it in the `DaughterStorage`. The further task of each manager is to calculate the Global trust of the system in its child node, and also to transfer all the information he has to other managers so that they, in turn, can perform a similar task.
+
+Global trust is calculated iteratively. In the process of calculating `GlobalTrust`, according to the EigenTrust algorithm, managers need to exchange “intermediate” trust values, the so-called `IntermediateTrust`. With the help of these values, managers at each iteration approximate (recalculate the results of the previous iteration) the value of the Global trust to the common Global value limit: $T = [t_0, ..., t_n]$, where $t_i$ is the Global trust value in the node `i` limit.
+
+All formulations, terms and statements in this clause take into account that Global trust makes sense only in the context of a certain epoch, that is, peer-to-peer interactions of nodes for the epoch `n` affect *only* the Global trust calculated in the epoch `n`. `IntermediateTrust` values, likewise, only make sense when used in the context of some epoch **and iteration number**. That fact is not emphasized explicitly further.
+
+The iterative formula:
+
+$$
+t_{j}^{k+1} =(1-\alpha) \sum_{j=0}^{n} C_{ij}t_i^k + \alpha t_j^0
+$$,
+
+where $C_{ij}$ is Local trust of the node `i` to the node `j`; $\alpha$ is a value that determines the influence of the $t_i^0$ - “blind” trust in the node `i` - on the final result; $C_{i,j}t_i^k$, in the introduced terminology, is `IntermediateTrust` which was transferred by the manager of the node `i` to the manager of the node `j`. In order for the algorithm to work synchronously, the number of iterations and $\alpha$ is read by all nodes at the beginning of each epoch from the global configuration of the `Netmap` contract.
+
+Blind trust $t_i^0$, generally speaking, can be different for different nodes. It can take into account how long ago a node joined the network, whether the node belongs to the developers of the network, etc. In the current implementation $t_i^0 = \frac{1}{N}, \forall i$, where $N$ is the number of network participants in the current epoch.
+
+### Transport
+
+In order for the manager of the node `j` to be able to calculate $t_j^{k+1}$ - the intermediate global trust (this is *not* `IntermediateTrust`) to its child node, it is necessary:
+
+1. The manager must have all the existing `IntermediateTrust`s - $C_{ij}t_i^k, \forall i \neq j$.
+2. The manager must calculate and transfer all `IntermediateTrust`s, according to the information received from its child node, that is, calculate and transfer all existing $C_{ji}t_j^k, \forall i \neq j$.
+
+To do this, the manager must act as both a client and a server. The transfer itself is done by calling the `AnnounceIntermediateResult` [gRPC method](https://github.com/nspcc-dev/neofs-api/blob/master/reputation/service.proto#L22).
```

Comment:
... act both as a client and a server.
The transfer itself is run...

Reply:
fixed